changelog.d/10062.removal

```diff
@@ -1 +1 @@
-The current spam checker interface is deprecated in favour of a new generic modules system. See the [upgrade notes](https://github.com/matrix-org/synapse/blob/master/UPGRADE.rst#deprecation-of-the-current-spam-checker-interface) for more information on how to update to the new system.
\ No newline at end of file
+The current spam checker interface is deprecated in favour of a new generic modules system. See the [upgrade notes](https://github.com/matrix-org/synapse/blob/master/docs/upgrade.md#deprecation-of-the-current-spam-checker-interface) for more information on how to update to the new system.
```

Comment:
This should match CHANGES.md

```suggestion
The current spam checker interface is deprecated in favour of a new generic modules system. See the [upgrade notes](docs/upgrade.md#deprecation-of-the-current-spam-checker-interface) for more information on how to update to the new system.
```

Reply:
Relative links won't work in the release notes page though I think.

Reply:
true, I guess.